src/datapoints-graph/model/datapoints-graph-widget.model.ts

```diff
@@ -162,3 +188,26 @@ export interface SeriesDatapointInfo {
   datapointLabel: string;
   datapointUnit: string;
 }
+
+export const SEVERITY_LABELS = {
+  CRITICAL: 'CRITICAL',
+  MAJOR: gettext('Major`alarm`') as 'MAJOR',
+  MINOR: gettext('Minor`alarm`') as 'MINOR',
+  WARNING: gettext('Warning`alarm`') as 'WARNING',
```

Comment:
```suggestion
  MAJOR: gettext('Major`alarm`') as 'Major',
  MINOR: gettext('Minor`alarm`') as 'Minor',
  WARNING: gettext('Warning`alarm`') as 'Warning',
```

Reply:
this is how it is implemented for the new alarms page, I would leave them as it is for consistency sake